content/SCALE/SCALETutorials/Apps/CommunityApps/WebDAV.md

```diff
@@ -1,109 +1,169 @@
+
+Enter a name in **Share Name**. 
+The name can have upper and lowercase letters and numbers. It can include an underscore (_) and/or a dash (-).
+
+Enter share use or other descriptive information about the share in **Description**. This is not required. 
+
+Enter or browse to the **Host Path** location for the where the app adds the WebDAV share. 
+If you created a dataset before installing the app, you can browse to it here. 
+
+Select **Read Only** to disable write access to the share.
 When selected, data accessed by clients cannot be modified.
 
-Set **Fix Permissions** to change the **Host Path** filesystem permissions.
-The dataset owner becomes the **User ID** and **Group ID** set in the **User and Group Configuration** section.
+Select **Fix Permissions** to change the **Host Path** file system permissions. 
+When enabled, the dataset owner becomes the **User ID** and **Group ID** set in the **User and Group Configuration** section.
 By default, this is the **webdav** user with UID and GID **666**.
+**Fix Permissions** allows TrueNAS to apply the correct permissions to the WebDAV shares and directories and simplify app deployment.
+After first configuration, the WebDAV container runs as the dedicated **webdav** user (UID: 666).
+
+{{< hint type=important >}}
+WebDAV only supports Unix-style permissions.
+When deployed with **Fix Permissions** enabled, it destroys any existing permissions scheme on the shared dataset.
+It is recommended to only share newly created datasets that have the **Share Type** set to **Generic**.
+{{< /hint >}}
+
+### Resources Configuration Settings
+By default, this application is limited to use no more than **4** CPU cores and **8** Gibibytes available memory.
+The application might use considerably less system resources.
 
-To specify a different user, enter the user ID (UID) and group ID (GID) in the fields for these settings.
-When migrating from an existing built-in WebDAV share, you might need to adjust the default to use whatever UID and GID was previously configured as the share path owner.
+{{< trueimage src="/images/SCALE/23.10/InstallWebDAVResourcesConfig.png" alt="WebDAV Resource Configuration" id="10: WebDAV Resource Configuration" >}}
 
-Click **Save** to save the configuration, deploy the app, and make any enabled **Shares** accessible.
+Tune these limits as needed to prevent the application from overconsuming system resources and introducing performance issues.
 
 ## Testing the Share
 
-At the end of this migration process, test access to your WebDAV share.
+At the end of the installation process, test access to your WebDAV share.
 
 In a browser, this is done by opening a new tab and entering the configured protocol, system host name or IP address, WebDAV port number, and **Share Name**.
-Example: `https://my-truenas-system.com:30001/mywebdavshare`
+Example: https://my-truenas-system.com:30001/mywebdavshare
```

Comment:
It might be a good idea to find a way to show this example that won't render an active (but non-working) hyperlink on the page

Reply:
This is why I put it in the code emphasis